redhat/overlays/Dockerfile

```diff
@@ -5,10 +5,13 @@ WORKDIR /cosign
 COPY . .
 USER root
 RUN git config --global --add safe.directory /cosign
-RUN make cosign
+RUN make cross
+RUN gzip cosign-darwin-amd64
+RUN gzip cosign-linux-amd64
+RUN gzip cosign-windows-amd64
 
 # Install Cosign
-FROM registry.access.redhat.com/ubi9/ubi-minimal@sha256:b40f52aa68b29634ff45429ee804afbaa61b33de29ae775568933c71610f07a4
+FROM registry.access.redhat.com/ubi9/ubi-minimal@sha256:0dfa71a7ec2caf445e7ac6b7422ae67f3518960bd6dbf62a7b77fa7a6cfc02b1
```

Comment:
Where is the `0dfa71` sha coming from? I still see `b40f52` as the latest here https://catalog.redhat.com/software/containers/ubi9/ubi-minimal/615bd9b4075b022acc111bf5?architecture=amd64&image=652fc5a903899c8ddcf105be

Reply:
Updated.